content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -382,33 +382,33 @@ An important remark about partial SRG blocks is that when the compiler, **AZSLc*
         OUT.m_color = PassSrg::GetColor();
         return OUT;
     }
-  
-## <a name="rootconstant"></a>Root Constants
+```
+## Root Constants
 AZSL supports the definition of shader constants that conform to the DX12 concept of Root Constants: https://docs.microsoft.com/en-us/windows/win32/direct3d12/using-constants-directly-in-the-root-signature.  
   
 Root Constants, declared with the `rootconstant` keyword, will be transparently embedded into a `ConstantBuffer`. They are useful to define shader constants of frequent access. Because they are located in the root signature they can be read instantly without requiring additional levels of indirection.  
```

Comment:
Can you verify the suggestion is accurate? Is AZSLc responsible for processing root constants as described here?
```suggestion
Root constants are declared with the `rootconstant` keyword. AZSLc transparently embeds root constants into a `ConstantBuffer`. They are useful to define shader constants that you frequently access. Because they are located in the root signature, AZSLc reads them instantly without requiring additional levels of indirection.  
```

Reply:
The only thing wrong is saying "AZSLc reads them". When the shader code runs inside the GPU it is the shader code itself who reads those constants.

Reply:
Okay sounds good. You can ignore the "AZSLc reads them" suggestion.